boards/arm/twr_ke18f/twr_ke18f.dts

```diff
@@ -122,6 +122,19 @@
 	};
 };
 
+&cpu0 {
+	clock-frequency = <120000000>;
+	cpu-power-states = <&idle &stop>;
+};
+
+&idle {
+	min-residency-us = <10>;
+};
+
+&stop {
+	min-residency-us = <100>;
+};
```

Comment:
@MaureenHelm I have been unable to locate any specifications of how long transitions to (partial) STOP mode takes, so these values are more or less randomly chosen for the development board.